api/src/main/resources/workbench-api.yaml

```diff
@@ -1480,10 +1480,32 @@ paths:
           schema:
             type: object
             required:
-              - value
+              - requestReason
+              - requestDate
             properties:
-              value:
-                type: boolean
+              requestReason:
+                type: string
+              requestDate:
+                type: string
```

Comment:
String-typed time/date fields are difficult to get right, and we've had some bugs related to this.  One example is https://precisionmedicineinitiative.atlassian.net/browse/RW-6952 

Instead, we should use an integer for date or time, and be explicit about what it means.  See some of the "epochMilli" fields for examples.